docs: add alternative snippet that blocks on sentry-cli errors (#5147)
getsentry/sentry-docs: src/includes/debug-symbols-apple/_default.mdx

1.  Download and install [sentry-cli](/product/cli/installation/)
2.  You will need to copy the script below into a new _Run Script_ and set your _AUTH_TOKEN_, _ORG_SLUG_, and _PROJECT_SLUG_

> <Alert level="" title="Warnings vs. errors">

Reviewer: Making this title case. Also added an empty line between the Alert tag and the markdown text. This ensures any formatting in your markdown renders properly.

```suggestion
<Alert level="" title="Warnings vs. Errors">

```

> Depending on your needs, you may want to emit warnings to the Xcode build log to let the build pass, or emit errors to fail it, like in a CI deployment where debug symbols may be harder to recover, and you might not realize symbols woudn't be able to be symbolicated until after release.

Reviewer: That sentence was too long, so I've broken it up

```suggestion
Depending on your needs, you may want to emit warnings to the Xcode build log to let the build pass, or emit errors to fail it. For example, in a CI deployment where debug symbols may be harder to recover, and you might not realize symbols couldn't be symbolicated until after release, it might be better to fail the build.
```

> Another option is to use warnings, and then set GCC_TREAT_WARNINGS_AS_ERRORS to YES in build settings for configurations you use to deploy to production.

Reviewer: ```suggestion
Another option is to use warnings, and then set `GCC_TREAT_WARNINGS_AS_ERRORS` to `YES` in build settings for configurations you use to deploy to production.

```
